sunpy/tests/helpers.py

```diff
@@ -159,79 +139,6 @@ def _patch_coverage(testdir, sourcedir):  # pragma: no cover
     cov.save()
 
 
-html_intro = '''
```

Comment:
Noice, did you put the HTML comparison stuff in pytest-mpl too?

Reply:
yep